src/buttons/FAB.tsx

```diff
@@ -83,7 +84,7 @@ const FAB: RneFunctionComponent<FABProps> = ({
         ])}
         titleStyle={[
           styles.label,
-          { color: theme.colors.white },
+          { color: theme && theme.colors && theme.colors.white },
```

Comment:
Might look nicer to use optional chaining: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining